code/modules/hippie/pool/pool_controller.dm

```diff
@@ -0,0 +1,363 @@
+
+//Originally stolen from paradise. Credits to tigercat2000.
+//Modified a lot by Kokojo.
+/obj/machinery/poolcontroller
+	name = "Pool Controller"
+	desc = "A controller for the nearby pool."
+	icon = 'icons/turf/pool.dmi'
+	icon_state = "poolcnorm"
+	anchored = 1
+	density = 1
+	use_power = 1
+	idle_power_usage = 75
+	var/list/linkedturfs = list() //List contains all of the linked pool turfs to this controller, assignment happens on New()
```

Comment:
Please set this to `null` and use `LAZYINITLIST` when you need to have it as a list, doing it this way adds hidden startup procs.

See [this](https://github.com/tgstation/tgstation/blob/4e640d1c3513bfe1d74a007fe6ca1f19c290c698/code/__HELPERS/lists.dm#L461) for all lazy helpers